File: C/unit_3/Fb.c
#include <stdio.h>
int main()
{
    int num;
    printf("Enter the Number : \n");
    scanf("%d", &num);
    if (num % 2 == 0)
        printf("The Number is an even number !!!");
    else
        printf("The number is an Odd number !!!!");
    return 0;
}

File: Java/unit_3/Fb.java
package Java.unit_3;

import java.util.Scanner;

public class Fb {
    public static void main(String[] args) {
        int num;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the Number : \n");
        num = sc.nextInt();
        if (num % 2 == 0)
            System.out.println("The Number is an Even Number !!!");
        else
            System.out.println("The Number is an odd Number !!!");
        sc.close();
    }
}
